### ISSUE TITLE
java.lang.ArrayIndexOutOfBoundsException: 0 on remote select after restore from backup

### ISSUE THREAD

**Issue by rinatio** [orientechnologies/orientdb#3662]
Hello guys,

I'm testing backup/restore with built in `GratefulDeadConcerts` database and getting a `java.lang.ArrayIndexOutOfBoundsException: 0` error on `select * from written_by` with remote connect after restore

Here're steps to reproduce:

```
# ./orientdb/bin/console.sh
orientdb> connect plocal:/opt/orientdb/databases/GratefulDeadConcerts admin admin
orientdb {db=GratefulDeadConcerts}> backup database /opt/orientdb/backup.zip
orientdb {db=GratefulDeadConcerts}> drop database
# sudo /etc/init.d/orientdb.sh start
orientdb> create database remote:127.0.0.1/GratefulDeadConcerts root password plocal
# /etc/init.d/orientdb.sh stop
orientdb {db=GratefulDeadConcerts}> connect plocal:/opt/orientdb/databases/GratefulDeadConcerts admin admin
orientdb {db=GratefulDeadConcerts}> restore database /opt/orientdb/backup.zip
orientdb {db=GratefulDeadConcerts}> quit
# /etc/init.d/orientdb.sh start
# ./orientdb/bin/console.sh
orientdb> connect remote:127.0.0.1/GratefulDeadConcerts admin admin
orientdb {db=GratefulDeadConcerts}> select * from written_by

Error: com.orientechnologies.orient.core.exception.OCommandExecutionException: Error on execution of command: sql.select * from written_by

Error: java.lang.ArrayIndexOutOfBoundsException: 0
```

Note: with local connect query doesn't throw this exception

**Comment by rinatio** [comment]
Is it safe to use backups with such exception? Will it be possible to restore the DB from such backup files once this issue is resolved?

**Comment by tglman** [comment]
merged in 2.0.x closing
